teamwork/templates/project_details.html

```diff
@@ -2,26 +2,55 @@
 {% load static %}
 
 {% block content %}
+<div id="page-wrapper">
+  <div class="container">
+    <!-- Project details -->
+    <h1>{{ project.name }} <br />
+      <small>{{ project.description }}</small></h1>
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title"><i class="glyphicon glyphicon-user"></i> Group member</h3>
+        </div>
+        <div class="panel-body">
+          <ul class="list-group">
+            {% for member in project.members.all %}
+            <li class="list-group-item">{{ member.user.username }}</li>
+            {% endfor %}
+          </ul>
+          <button type="button" class="btn btn-default" onclick="openEventCreator()">Add member</button>
+        </div>
+      </div>
 
-<h1>{{ project.name }}</h1>
+      <!-- Time Spent -->
+      <h3>Time Spent</h3>
+      <!-- Logged time history from all the project memebers -->
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <h3 class="panel-title"><i class="glyphicon glyphicon-time"></i> Log history</h3>
+        </div>
+        <div class="panel-body">
+          <div class="list-group">

```

Comment:
For lists, it would be semantically better to use `<ul>` and `<li>` tags. UL means unordered list and LI means list item.